### DISCUSSION TITLE
Compile error on branch master.

### DISCUSSION THREAD

**Discussion by SummerGaoPlus** [emqx/emqx#10832] [General]
### What happened?

![image](https://github.com/emqx/emqx/assets/32028756/930d4570-4d42-4a0f-9cb1-9f523abe53a5)

### What did you expect to happen?

![image](https://github.com/emqx/emqx/assets/32028756/3dbdf356-6cd2-4bef-a129-5191283d2098)

### EMQX version

master

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Log files

<details>

</details>

**Comment by zhongwencool** [comment]
This error mean can't find `hocon/include/hoconsc.hrl`, please make sure the hocon lib is in _build/default/lib/ 
This simplest way: Try `make clean-all` and `make ` again...

**Reply by SummerGaoPlus** [reply]
ok
